packages/documentation/src/stories/getting-started/design-principles/design-principles.docs.mdx

```diff
@@ -0,0 +1,25 @@
+import { Meta } from '@storybook/blocks';
+import * as DesignPrinciplesStories from './design-principles.stories';
+import './design-principles.styles.scss';
+import { principles } from './design-principles.data';
+
+<Meta of={DesignPrinciplesStories} />
+
+# Swiss Post Design Principles
+
+Our aim is to make things simple, trustworthy, and inclusive. We create easy-to-use designs, build trust with clear communication, and ensure everyone feels included.
+
+## Core Principles
+
+<div className="row g-4">
+  {principles.map((principle, index) => (
+    <div className="col-lg-3 d-flex align-items-stretch" key={index}>
+      <div className="principle p-4 border h-100">
+        <post-icon name={principle.icon} style={{ fontSize: '3rem' }}></post-icon>
+        <h3 className="principle--title">{principle.title}</h3>
```

Comment:
Just a small suggestion. Otherwise, the cards look great!
```suggestion
        <h3 className="principle--title mt-small-large">{principle.title}</h3>
```